src/po/fi.po

```diff
@@ -575,8 +576,8 @@ msgstr "Takaperoinen arvoalue annettu, OK kääntää"
 msgid ""
 "INTERNAL: Cannot use EX_DFLALL with ADDR_NONE, ADDR_UNSIGNED or ADDR_QUICKFIX"
 msgstr ""
-"Sisäinen: Ei voi käyttää EX_DFLALLia yhdessä ADDR_NONEn, ADDR_UNSIGNEDin tai"
-"ADDR_QUICKFIXin kanssa"
```

Comment:
colon is not used in Finnish in inflecting full words like this, see http://www.kielitoimistonohjepankki.fi/ohje/278